libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectDistanceInfoType.kt

```diff
@@ -0,0 +1,54 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import androidx.annotation.IntDef
+
+/**
+ * Holds available [RoadObjectDistanceInfo] types.
```

Comment:
Could you describe high level what distance info is?

Reply:
done